lib/grover/js/processor.js

```diff
@@ -190,6 +202,23 @@ const _processPage = (async (convertAction, urlOrHtml, options) => {
       await page.hover(hoverSelector);
     }
 
+    if (errors.length > 0) {
+      function RequestFailedError(errors = []) {
+        this.name = "RequestFailedError";
+        this.message = errors.map(e => {
+          if(e.failure()) {
+            return e.failure().errorText + " at " + e.url();
+          } else if (e.response() && e.response().status()) {
+            return e.response.status() + " " + e.url();
+          } else {
+            return "ERR" + " " + e.url()
```

Comment:
Two things here. 
1. "ERR" seems maybe a tad vague. It seems like timeouts etc would be handled by the `e.failure()` case and bad responses by the `e.response().status()` case. What do you see this handling? If it's "unknown" then suggest using language to that point.
2. `"ERR " + e.url()` would seem simpler (albeit after addressing point 1.)

Reply:
I put this in before the code was properly handling the previous cases. I think the way the code is structured, I would prefer to keep a `else` condition (so we don't have things in the `errors` array that don't have error messages) so I'll change this to state it's an unknown error.